### DISCUSSION TITLE
uWS and stateful architecture

### DISCUSSION THREAD

**Discussion by An88ton** [uNetworking/uWebSockets.js#551] [Q&A] [answered]
Hi, I have a project on standart js websockets, and want to move it on uws, but have a problem, each one websocket should be bind with other object, is there ways to do it with uws( search in arrays is not appropriate, because of high load of service)

**Comment by ghost** [comment, marked as answer]
The JS object you get in open is the same you get in all other events

**Reply by An88ton** [reply]
Thank u for fast answer, you mean I can add field to ws object in open, and it will be ok, right?
